tests/test_project/tests/test_selenium.py

```diff
@@ -0,0 +1,418 @@
+            self.assertEqual(mg_dropdown.is_displayed(), False)
+            # Test menu dropdown gets invisible when clicked outside on wide screen
+            if not is_medium and not is_narrow:
+                mg_head.click()  # Show dropdown
+                main_content.click()
+                self.assertEqual(mg_dropdown.is_displayed(), False)
+
+        if is_narrow:
+            # Do not test when menu is not visible
+            return
+        with self.subTest('Test menu dropdown when menu is close'):
+            self._close_menu()
+            # Test mg_dropdown is not visible
+            self.assertEqual(mg_dropdown.is_displayed(), False)
+            self.assertEqual(mg_label.is_displayed(), False)
+            self.assertEqual(mg_icon.is_displayed(), True)
+            # Test mg dropdown gets visible on clicking mg head
+            mg_head.click()
+            is_visible = True
+            try:
+                WebDriverWait(self.web_driver, 2).until(
+                    EC.visibility_of_element_located(
+                        (By.XPATH, '//*[@class="nav"]/div[1]/div[2]')
+                    )
+                )
+            except TimeoutException:
+                is_visible = False
+            self.assertEqual(
+                is_visible, True,
+            )
+            self.assertEqual(mg_dropdown_label.is_displayed(), True)
+            # Test mg dropdown gets invisible on clicking mg head
+            mg_head.click()
+            self.assertEqual(mg_dropdown.is_displayed(), False)
+            # Test menu dropdown gets invisible when clicked outside
+            mg_head.click()  # Show dropdown
+            main_content.click()
+            self.assertEqual(mg_dropdown.is_displayed(), False)
+
+        with self.subTest('Test visibilty of menu label when menu close'):
+            self._close_menu()
+            actions = ActionChains(self.web_driver)
+            actions.move_to_element(mg_head)
+            actions.perform()
+            is_visible = True
+            try:
+                WebDriverWait(self.web_driver, 2).until(
+                    EC.visibility_of_element_located(
+                        (By.XPATH, '//*[@class="nav"]/div[1]/div[1]/div[1]/span[2]')
+                    )
+                )
+            except TimeoutException:
+                is_visible = False
+            self.assertEqual(is_visible, True)
+            mg_head.click()
+            actions.move_to_element(mg_head)
+            actions.perform()
+            self.assertEqual(mg_label.is_displayed(), False)
+
+    def _test_popup_page(self):
+        self.open('/admin/auth/user/add/?_to_field=id&_popup=1')
+        WebDriverWait(self.web_driver, 2).until(
+            EC.visibility_of_element_located(
+                (By.XPATH, self.config['main_content_xpath'])
+            )
+        )
+        with self.assertRaises(NoSuchElementException):
+            self._get_menu()
+        self.open('/admin/')
```

Comment:
same here and any other occurrence, please use `reverse()` which makes the code resistant to changes.